### PR TITLE
Update new API pool farm

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
This PR is part of refactoring process that is to use our own BE to handle pool/farm data. The next release will only affect Avalanche and Optimism, so there're 2 versions co-existing: v1 for old version and v2 for the version in this PR.

This PR only changes the source to get pool/farm data from SubGraph to our BE. Almost 99% of the logic remains the same.

I added `internalRoute` to use for our internal API.

### CODE REVIEW

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/pages/ProAmmPools/ListItem.tsx [hunk 1]
```diff
@@ -148,6 +147,43 @@ export default function ProAmmPoolListItem({ pair, idx, onShared, userPositions,
...
+            }
+
+            return <FarmingPoolAPRCell poolAPR={pool.apr} fairlaunchAddress={fairlaunchAddress} pid={pid} />
+          }
+
+          return (
+            <Flex
+              sx={{
+                alignItems: 'center',
+                paddingRight: '20px', // to make all the APR numbers vertically align
```
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/pages/ProAmmPools/ListItem.tsx]: Yeah I fixed

File: src/constants/networks/avax-testnet.ts [hunk 2]
```diff
@@ -59,6 +59,7 @@ const avaxTestnetInfo: NetworkInfo = {
   coingeckoNetworkId: EMPTY,
   coingeckoNativeTokenId: EMPTY,
   deBankSlug: EMPTY,
+  internalRoute: EMPTY,
```
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/constants/networks/avax-testnet.ts:62]: BE said they don't support test net so I still leave this empty

File: src/constants/networks/velas.ts [hunk 3]
```diff
@@ -63,6 +63,7 @@ const velasInfo: NetworkInfo = {
   coingeckoNetworkId: 'velas',
   coingeckoNativeTokenId: 'velas',
   deBankSlug: EMPTY,
+  internalRoute: EMPTY,
```
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/constants/networks/velas.ts]: Fixed

File: src/constants/networks/fantom.ts [hunk 4]
```diff
@@ -59,6 +59,7 @@ const fantomInfo: NetworkInfo = {
   coingeckoNetworkId: 'fantom',
   coingeckoNativeTokenId: 'fantom',
   deBankSlug: 'ftm',
+  internalRoute: EMPTY,
```
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/constants/networks/fantom.ts]: Fixed

File: src/constants/networks/aurora.ts [hunk 5]
```diff
@@ -60,6 +60,7 @@ const auroraInfo: NetworkInfo = {
   coingeckoNetworkId: 'aurora',
   coingeckoNativeTokenId: 'ethereum',
   deBankSlug: 'aurora',
+  internalRoute: EMPTY,
```
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/constants/networks/aurora.ts]: Fixed

File: .env.dev [hunk 6]
```diff
@@ -36,3 +36,5 @@ REACT_APP_KS_SETTING_API=https://ks-setting.dev.kyberengineering.io/api
 REACT_APP_GTM_ID=
 
 REACT_APP_GOOGLE_RECAPTCHA_KEY=6Lc7yg0iAAAAALhS4C4Ez5hqMBub8hKdFvvr9sfc
+
+REACT_APP_POOL_FARM_BASE_URL=https://pool-farm.kyberswap.com
```
Comment (namgold) [inline .env.dev:40]: missed update this for .env.stg

File: src/constants/networks/aurora.ts [hunk 7]
```diff
@@ -53,13 +53,14 @@ const auroraInfo: NetworkInfo = {
     nonfungiblePositionManager: '0x2B1c7b41f6A8F2b2bc45C3233a5d5FB3cD6dC9A8',
     tickReader: '0x165c68077ac06c83800d19200e6E2B08D02dE75D',
     initCodeHash: '0xc597aba1bb02db42ba24a8878837965718c032f8b46be94a6e46452a9f89ca01',
-    quoter: '0x0D125c15D54cA1F8a813C74A81aEe34ebB508C1f',
+    quoter: '0x0D125c15D54cA1F8aw813C74A81aEe34ebB508C1f',
```
Comment (viet-nv) [inline src/constants/networks/aurora.ts:56]: why do you change this?

File: .env.dev (hunk 6 above)
Comment (hungdoansy) [reply .env.dev:40]: They don't have API for staging. Only prod

Comment (hungdoansy) [reply .env.dev:40]: Good catch. Thanks

File: src/constants/networks/avax-testnet.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/constants/networks/avax-testnet.ts:62]: so please add production api on staging env, then UI can load farm normally on staging

File: src/constants/networks/aurora.ts (hunk 7 above)
Comment (hungdoansy) [reply src/constants/networks/aurora.ts]: My bad. This is a typo. I removed

**Review by viet-nv — APPROVED** [review]

**Pull request merged**